lib/camel_patrol/middleware.rb

```diff
@@ -17,14 +17,17 @@ def call(env)
     def underscore_params(env)
       if ::Rails::VERSION::MAJOR < 5
         env["action_dispatch.request.request_parameters"].deep_transform_keys!(&:underscore)
-      else
-        request_body = JSON.parse(env["rack.input"].read)
-        request_body.deep_transform_keys!(&:underscore)
-        req = StringIO.new(request_body.to_json)
+        return
```

Comment:
is this `return` necessary?

Reply:
Yeah, because I've replaced the if/elseif clause with a guard for the initial `if` criteria

Reply:
oh, derp. i see that now.